src/server.ts

```diff
@@ -1,9 +1,4 @@
 /* istanbul ignore file */
-
-if (process.env.NEW_RELIC_ENABLED !== 'false') {
```

Comment:
how will it get executed in production, which does not use npm scripts?

Reply:
Could add it to `CMD` in `Dockerfile`.

Reply:
that would be difficult to delete for template users who don't use New Relic, as that will not show up during TS compilation, and can cause lots of headache.